UOP1_Project/Assets/Scripts/Editor.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: de5c57e7db463f74a91926f18b81eb2e
```

Comment:
I don't think this change should have happened.. Seems both unrelated and kinda dangerous that it somehow changes the guid of the editor folder. Please revert this

Reply:
I've restored the missing files by cherrypicking